user.go

```diff
@@ -251,6 +251,16 @@ func (user *User) startupTryConnect(retryCount int) {
 		} else {
 			user.BridgeState.Send(status.BridgeState{StateEvent: status.StateUnknownError, Error: "dc-unknown-websocket-error", Message: err.Error()})
 		}
+	} else {
+		user.updatePuppetContactInfo()
+	}
+}
+
+func (user *User) updatePuppetContactInfo() {
+	if user.bridge.Config.Homeserver.Software == bridgeconfig.SoftwareHungry {
+		for _, puppet := range user.bridge.GetAllPuppets() {
+			puppet.UpdateContactInfo(user, nil)
```

Comment:
This looks like it'd spam too many requests if there are big guilds bridged

Reply:
Probably just need to let it sync as messages come in. DMs will also get synced on startup.